pyls/python_ls.py

```diff
@@ -39,8 +40,10 @@ def capabilities(self):
             'textDocumentSync': lsp.TextDocumentSyncKind.INCREMENTAL
         }
 
-    def initialize(self, root_path, init_opts, _process_id):
-        self.workspace = Workspace(root_path, lang_server=self)
+    def initialize(self, root_uri, init_opts, _process_id):
+        self.workspace = Workspace(root_uri, lang_server=self)
+
+        root_path = unquote(urlparse(root_uri).path)
         self.config = config.Config(root_path, init_opts)
```

Comment:
Thanks for the clarification, got a chance to implement your suggestions just now!